http/sys_ha_status.go

```diff
@@ -0,0 +1,68 @@
+package http
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/vault/vault"
+	"github.com/shirou/gopsutil/host"
+)
+
+func handleSysHaStatus(core *vault.Core) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			handleSysHaStatusGet(core, w, r)
+		default:
+			respondError(w, http.StatusMethodNotAllowed, nil)
+		}
+	})
+}
+
+func handleSysHaStatusGet(core *vault.Core, w http.ResponseWriter, r *http.Request) {
+	_, address, clusterAddr, err := core.Leader()
+	if errwrap.Contains(err, vault.ErrHANotEnabled.Error()) {
+		err = nil
+	}
+	if err != nil {
+		respondError(w, http.StatusInternalServerError, err)
+		return
+	}
+	h, _ := host.Info()
+	nodes := []Node{
+		{
+			Hostname:       h.Hostname,
```

Comment:
Yeah, that's left over from the original PR, will rename.

Reply:
Actually it's more complicated than that.  NodeID was a pre-existing field used only in the replication RPCs, unpopulated in the perf standby RPCs.  I think instead I'll introduce a new field to avoid confusion.

Reply:
Done.